tasks/main.yml

```diff
@@ -74,15 +74,24 @@
     key: https://packages.microsoft.com/keys/microsoft.asc
     state: present
 
+# This works only on systems using yum or dnf
+- name: Update all packages from SQL Server 2017 repo
+  package:
+    name: "*"
+    disablerepo: "*"
+    enablerepo: packages-microsoft-com-mssql-server-2017
+    state: latest  # noqa E403
```

Comment:
Looks like this isn't working to suppress the ansible-lint warning

Reply:
Fixed with `# noqa 403`